keyboards/kelwin46.5/kelwin46.5/info.json

```diff
@@ -0,0 +1,114 @@
+{
+    "keyboard_name": " kelwin46.5",
+    "manufacturer": "kelwin",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x4B57",
+        "pid": "0x0000",
+        "device_version": "1.0.0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "rows": ["F4", "F1", "F6", "F7", "C7"],
+        "cols": ["F0", "F5", "B0", "B7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D0", "D3", "D2", "D1"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "all": true
+        "alternating": true,
+        "breathing": true,
+        "christmas": true,
+        "knight": true,
+        "rainbow_mood": true,
+        "rainbow_swirl": true,
+        "rgb_test": true,
+        "snake": true,
+        "static_gradient": true,
+        "twinkle": true
+        }

```

Comment:
If you meant to remove the rgb config, you need to remove this section, as well:
```suggestion
```